### PR TITLE
Update all kubectl run commands (now deprecated)

### PR DESCRIPTION
**Pull request by evilnick**
This only seems of concern in the integrator page examples. This PR fixes the examples in the general pages, the master charm docs and the relevant versioned charm pages.

### CODE REVIEW

File: pages/k8s/1.18/charm-gcp-integrator.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ The following script starts the hello-world pod behind a GCE-backed load-balance
 ```sh
 #!/bin/bash
 
-kubectl run hello-world --replicas=5 --labels="run=load-balancer-example" --image=gcr.io/google-samples/node-hello:1.0  --port=8080
+kubectl create deployment hello-world --image=gcr.io/google-samples/node-hello:1.0 --replicas=5 --port=8080
 kubectl expose deployment hello-world --type=LoadBalancer --name=hello
 watch kubectl get svc -o wide --selector=run=load-balancer-example
```
Comment (evilnick) [inline pages/k8s/1.18/charm-gcp-integrator.md:151]: bugger - this won't work either will it, it needs to point to the app

**Comment by evilnick** [comment]
@johnsca I fixed all the `get svc` examples now. Did we come to some conclusion on the versioning/not versioning ?

**Comment by johnsca** [comment]
@evilnick I updated the first example to be version agnostic by doing the scale as a separate command and moving the `--port` flag to the `kubectl expose` call.

**Comment by evilnick** [comment]
@johnsca I replicated that change across all the relevant charm versions now. Is it ready for a :+1: ?

**Review by johnsca — APPROVED** [review]

**Comment by evilnick** [comment]
looks good. thanks!

**Pull request merged**